src/ExpressReceiver.ts

```diff
@@ -104,6 +106,8 @@ export default class ExpressReceiver implements Receiver {
         installationStore,
         stateStore: installerOptions.stateStore,
         authVersion: installerOptions.authVersion!,
```

Comment:
Why is `installerOptions.authVersion` the only one to get the non-null assertion?  ie, `stateStore` also defaults

Reply:
lol the short answer is that the editor yelled at me and i wanted it to stop. 

The real answer is that the type of `installerOptions.authVersion` can only be `'v1' | 'v2'` and in this line, it could potentially be `undefined`. Even if it is `undefined` here, once it makes it to `@slack/oauth`, it will get the default value of `v2` assigned if that is the case.